src/dfChemistryModel/dfChemistryModel.H

```diff
@@ -222,12 +222,14 @@ public:
 
         const volScalarField& rhoD(const label i) const {return rhoD_[i];}
 
-        volScalarField& hai(const label i) {return hai_[i];}
+        const volScalarField& hai(const label i) {return hai_[i];}
```

Comment:
was there a reason for not being const before?

Reply:
just a bad programing style